crates/ruff_workspace/src/configuration.rs

```diff
@@ -603,6 +627,18 @@ impl Configuration {
             );
         }
 
+        for selection in deprecated_nursery_selectors {
+            let (prefix, code) = selection.prefix_and_code();
+            warn_user!("Selection of nursery rule `{prefix}{code}` without the `--preview` flag is deprecated.",);
+        }
```

Comment:
The redirect warning code has a comment that says this should be in the `ruff_cli` crate — how would we go about that? I'd like to create a tracking issue.

Reply:
I'd vote to just remove that TODO -- I can't remember why it's there.